src/scss/components/_heroSlide.scss

```diff
@@ -0,0 +1,40 @@
+.slider{
+  margin: 0 auto;
+  position: relative;
+  max-width: 1440px;
+  &__img{
+    display: flex;
+    flex-direction: row-reverse;
```

Comment:
add `width: 100%;` to make sure this element will expand across all the container.